ServerHooks.cpp

```diff
@@ -140,19 +140,23 @@ void Eluna::OnChange(Weather* /*weather*/, uint32 zone, WeatherState state, floa
 // Auction House
 void Eluna::OnAdd(AuctionHouseObject* /*ah*/, AuctionEntry* entry)
 {
+#ifdef AZEROTHCORE
+    Player* owner = eObjectAccessor()FindPlayer(entry->owner);
+#else
     Player* owner = eObjectAccessor()FindPlayer(MAKE_NEW_GUID(entry->owner, 0, HIGHGUID_PLAYER));
+#endif
+
 #ifdef TRINITY
     Item* item = eAuctionMgr->GetAItem(entry->itemGUIDLow);
     uint32 expiretime = entry->expire_time;
 #elif AZEROTHCORE
-    Item* item = eAuctionMgr->GetAItem(entry->item_guidlow);
+    Item* item = eAuctionMgr->GetAItem(entry->item_guid);
```

Comment:
guid thing here too

Reply:
This is how it is done in AzerothCore and that line is only for AzerothCore.